app/src/ui-blokada/res/values/strings_bits.xml

```diff
@@ -14,7 +14,7 @@
     <string name="home_dns_only">Blokada is active in DNS mode</string>
 
     <!-- A text on the main ads counter. "490 requests blocked" -->
-    <string name="home_requests_blocked">%s blocked</string>
+    <string name="home_requests_blocked">%s blocked since %s</string>
```

Comment:
Please introduce a new string for that. Changing the number of placeholders in a preexisting string, messes up the existing translations and afair leads to crashes

Reply:
Is it sufficient to rename it or does the old string still has to exist?

Reply:
Old string does not have to exist.